net-misc/udpxy/files/udpxy-1.0.23.7-cflags.patch

```diff
@@ -0,0 +1,12 @@
+https://bugs.gentoo.org/725238
+--- a/Makefile
++++ b/Makefile
+@@ -100,7 +100,7 @@ CORES := core.* core
+ debug ldebug rdebug lean verify touch incbuild 
+ 
+ .c.o :
+-	$(CC) $(ALL_CFLAGS) $(CPPFLAGS) $(CDEFS) $(COPT) -c $< -o $@
++	$(CC) $(ALL_CFLAGS) $(CFLAGS) $(CPPFLAGS) $(CDEFS) $(COPT) -c $< -o $@
```

Comment:
`ALL_CFLAGS` doesn't contain weird crap?

Reply:
Nope, it seems to be empty, but I've dropped it now nevertheless